SwiftKitCore/build.gradle

```
@@ -0,0 +1,73 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift.org project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift.org project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+plugins {
+    id("build-logic.java-application-conventions")
+}
+
+group = "org.swift.swiftkitcore"
```

So the group for them all is `org.swift.swiftkit`

and the artif

```suggestion
group = "org.swift.swiftkit"
```

but the artifact id will come from the module name, which will be `SwiftKitCore` and `SwiftKitFFM` -> `swiftkit-core` + `swiftkit-ffm`

Setting those is done in `publishing { publications { maven(MavenPublication)` if I read this correctly... Either way -- the group ID remains the same for all those and you don't need to worry about the artifact IDs yet